Misc/NEWS.d/next/IDLE/2019-08-21-16-02-49.bpo-37902._R_adE.rst

```diff
@@ -0,0 +1 @@
+Add scrolling for IDLE browsers.
```

Comment:
It already has up/down arrow and page up/down key scrolling and scrollbar scrolling.
```suggestion
Add mousewheel scrolling for IDLE class, path, and stack browsers.
```

Reply:
Should this say "module" browser, not "class" browser? (Lingering memories eh?)

Reply:
Yes, and yes.